### DISCUSSION TITLE
edit block actions

### DISCUSSION THREAD

**Discussion by delkawy** [yoopta-editor/Yoopta-Editor#359] [Q&A]
how can I edit Block Actions?

**Comment by Darginec05** [comment]
Hi @delkawy 
What do you mean by "how can I edit Block Actions"?

**Reply by delkawy** [reply]
![image](https://github.com/user-attachments/assets/a12f66bf-4221-46db-9ed4-41d2745c6a03)

this action menu

**Reply by Darginec05** [reply]
@delkawy Oh yes, I've been planning to add the ability to customize this component for a long time.
So, converting it to to issue
